app/models/article.rb

```diff
@@ -648,13 +644,16 @@ def evaluate_markdown
   end
 
   def extracted_evaluate_markdown
-    return unless processed_content
+    content_renderer = processed_content
```

Comment:
Caching in a variable so that the code wouldn't run multiple times. I decided that the `processed_content` method is still useful.